modules/packages/Image.chpl

```diff
@@ -357,4 +370,299 @@ module Image {
     }
   }
 
+  private module BMPHelper {
+
+    enum Implementation {
+      native,
+      stb_image
+    }
+    config param impl = Implementation.native;
+    proc write(outfile, pixels: [?d]) throws {
+      if impl == Implementation.native {
+        writeNative(outfile, pixels);
+      } else if impl == Implementation.stb_image {
+        compilerError("std_image is not yet supported for BMP");
```

Comment:
Either `stb_image` or `std_image` is wrong here, probably the former.

Reply:
It should be `stb_image`, fixed!

Reply:
I meant "latter" 🤦